website/versioned_docs/version-5.x/ecosystem-react.md

```diff
@@ -96,7 +96,12 @@ You can use the Parcel component either by npm installing the library and import
 
 #### Examples
 ```jsx
+// webpack 5 syntax
 import Parcel from 'single-spa-react/parcel'
+// webpack 4 syntax
```

Comment:
webpack4 / rollup / others?

Reply:
Anything that doesn't support package.json expors. See related https://github.com/single-spa/single-spa-react/releases/tag/v3.0.0